### PR TITLE
recently_updated_ui

### PR DESCRIPTION
**Pull request by ckmah**

### CODE REVIEW

**Comment by ckmah** [comment]
ui for issue #64

**Comment by williamxu** [comment]
Does the unread indicator # update after reading? When I checked it remained at the previous number until the next time I refreshed the activity.

**Comment by williamxu** [comment]
When all manga chapters have been read, the indicator incorrectly displays 99+ instead of 0 or disappearing.

File: app/src/main/java/ckmah/mangoreader/database/Chapter.java [hunk 1]
```diff
@@ -4,7 +4,7 @@
     public String id;
     public String title;
     public String number; //LOL yes, number is a string. something to do with titles.
-    public double date;
+    public long date;
```
Comment (williamxu) [inline app/src/main/java/ckmah/mangoreader/database/Chapter.java:7]: Wait, it's a Long? I thought the response from MangaEden returned a double

**Comment by ckmah** [comment]
"indicator incorrectly displays 99+ instead of 0 or disappearing" is same bug as indicator # not updating after reading

File: app/src/main/java/ckmah/mangoreader/database/Chapter.java (hunk 1 above)
Comment (ckmah) [inline app/src/main/java/ckmah/mangoreader/database/Chapter.java:7]: no sir, definitely a long. unix style time stamp

**Pull request merged**